src/lib/viewers/image/ImageBaseViewer.js

```diff
@@ -434,6 +435,42 @@ class ImageBaseViewer extends BaseViewer {
         }
     }
 
+    /**
+     * Prints image using an an iframe.
+     *
+     * @return {void}
+     */
+    print() {
+        const browserName = Browser.getName();
+
+        /**
+         * Called async to ensure resource is loaded for print preview. Then removes listener to prevent
+         * multiple handlers.
+         *
+         * @return {void}
+         */
+        const defaultPrintHandler = () => {
+            if (browserName === 'Explorer' || browserName === 'Edge') {
+                this.printframe.contentWindow.document.execCommand('print', false, null);
+            } else {
+                this.printframe.contentWindow.print();
+            }
+
+            this.printframe.removeEventListener('load', defaultPrintHandler);
+            this.emit('printsuccess');
+        };
+
+        this.printframe = util.openContentInsideIframe(this.imageEl.outerHTML);
+        this.printImages = this.printframe.contentDocument.querySelectorAll('img');
+        this.printImages.forEach(element => {
+            element.setAttribute('style', 'display: block; margin: 0 auto; width: 100%');
+        });
+
+        this.printframe.contentWindow.focus();
+
+        this.printframe.addEventListener('load', defaultPrintHandler);
```

Comment:
In the original function from ImageViewer.js, the `focus` and `addEventListener` calls occurred right after the call to `openContentInsideIframe`. Will the reordering of these calls change the behavior of this function at all?